runtime/server/generate_metrics_view.go

```diff
@@ -447,6 +448,7 @@ type metricsViewYAML struct {
 }
 
 type metricsViewDimensionYAML struct {
+	Name        string `yaml:"name"`
```

Comment:
Are you sure we want to include `name` here? The default behavior is that it's the same as `column`, which is simpler and sufficient for most dimensions

Reply:
Ya for consistency it makes sense. While it is not needed it will be good for users to get this while autogenerated.

Reply:
Tagging @ericpgreen2 who asked for it for more clarity.

Reply:
Yeah I think we should populate it – because the Visual Metrics Editor considers `name` a required field in the Add/Edit Dimension form.

Reply:
Sounds like that needs to be fixed regardless in the visual editor then? Since there are many metrics views with only a `column:` field for dimensions?

Also, what's the workflow here for people who are adding/changing a dimension that maps directly to an underlying column? Do they have to enter/update the column in two places to get the expected behavior where the dimension name matches the underlying column name?

Reply:
We will be making `name` mandatory during parsing as well. Adding it during autogeneration is the 1st step